src/vsr/replica.zig

```diff
@@ -5336,24 +5509,27 @@ pub fn ReplicaType(
                 }
 
                 self.ping_timeout.start();
-                self.commit_timeout.stop();
-                self.normal_status_timeout.start();
+                self.commit_message_timeout.stop();
+                self.normal_heartbeat_timeout.start();
+                self.start_view_change_window_timeout.start();
+                self.start_view_change_message_timeout.start();
                 self.view_change_status_timeout.stop();
                 self.view_change_message_timeout.stop();
                 self.repair_timeout.start();
             }
 
+            self.heartbeat_timestamp = 0;
+            self.primary_abdicating = false;
```

Comment:
This can be assert I think? `primary_abdicating` can't be true if `.status = .view_change`, and it was view_chaeg.